source/indexes.txt

```diff
@@ -0,0 +1,278 @@
+.. guide.
+
+Wildcard Index
+--------------
+
+The following example creates an ascending wildcard index on the
+collection:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.php
+   :start-after: start-wildcard
+   :end-before: end-wildcard
+   :language: php
+   :copyable:
+   :dedent:
+
+.. TODO: To learn more about wildcard indexes, see the :ref:`php-wildcard-index`
+.. guide.
+
+Clustered Index
+---------------
+
+You can create a clustered index when creating a new collection in a
+specified database. The following example creates a new collection with an
+ascending clustered index on the ``_id`` field:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.php
+   :start-after: start-clustered
+   :end-before: end-clustered
+   :language: php
+   :copyable:
+   :dedent:
+
+.. TODO: To learn more about clustered indexes, see the :ref:`php-clustered-index`
+.. guide.
+
+Atlas Search Index Management
+-----------------------------
+
+The following sections contain code examples that describe how to manage
+:atlas:`Atlas Search indexes </atlas-search/manage-indexes/>`. 
+
+.. To learn more about Atlas Search indexes, see the :ref:`php-atlas-search-index`
+.. guide.
+
+Create Search Index
+~~~~~~~~~~~~~~~~~~~
+
+The following example creates an Atlas Search index on the specified field:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.php
+   :start-after: start-search-create
+   :end-before: end-search-create
+   :language: php
+   :copyable:
+   :dedent:
+
+.. To learn more about creating search indexes, see the
+.. :ref:`php-atlas-search-index-create` guide. 
+
+List Search Indexes
+~~~~~~~~~~~~~~~~~~~
+
+The following example prints a list of Atlas Search indexes in the
+specified collection:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.php
+   :start-after: start-search-list
+   :end-before: end-search-list
+   :language: php
+   :copyable:
+   :dedent:
+
+.. To learn more about listing search indexes, see the :ref:`php-atlas-search-index-list`
+.. guide.
+
+Update Search Indexes
+~~~~~~~~~~~~~~~~~~~~~
+
+The following example updates an existing Atlas Search index with the specified
+new index definition:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.php
+   :start-after: start-search-update
+   :end-before: end-search-update
+   :language: php
+   :copyable:
+   :dedent:
+
+.. To learn more about updating search indexes, see the :ref:`php-atlas-search-index-update`
+.. guide.
+
+Delete Search Indexes
+~~~~~~~~~~~~~~~~~~~~~
+
+The following example deletes an Atlas Search index with the specified name:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.php
+   :start-after: start-search-delete
+   :end-before: end-search-delete
+   :language: php
+   :copyable:
+   :dedent:
+
+.. To learn more about deleting search indexes, see the :ref:`php-atlas-search-index-drop`
+.. guide.
+
+Text Index
```

Comment:
I find it odd that the Atlas Search Index Management section appears in the middle of this page and is flanked by other sections that apply to non-search indexes. IMO, it'd be more clear to the reader to segregate all search and non-search index content. This can either be done with a separate heading for the non-search indexes, or just by putting Atlas Search Index Management at the very end.

This wasn't so much an issue with Text Index as it was with the List and Delete sections that follow. It seemed inconsistent to not present List and Delete after the Create examples, as was the natural case within the Atlas Search Index Management section.